templates/web/northnorthants/tracking_code.html

```diff
@@ -0,0 +1,94 @@
+[% IF c.config.BASE_URL == "https://www.fixmystreet.com" OR c.config.BASE_URL == 'https://staging.fixmystreet.com' %]
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag() {
+    dataLayer.push(arguments);
+  }
+
+  window.onload = () => {
+    const cookieBanner = document.getElementById("wncCookieBanner");
+    if (localStorage.getItem("wncCookieConsentMode") === null) {
+      cookieBanner.classList.remove("wnc-cookie-hidden");
+      cookieBanner.classList.add("wnc-cookie-show");
+    }
+
+    if (localStorage.getItem("wncCookieConsentMode") === null) {
+      gtag("consent", "default", {
+        ad_storage: "denied",
+        analytics_storage: "denied",
+        personalization_storage: "denied",
+        functionality_storage: "denied",
+        security_storage: "denied",
+      });
+    } else {
+      console.log('gtag');
+      gtag(
+        "consent",
+        "default",
+        JSON.parse(localStorage.getItem("wncCookieConsentMode"))
+      );
+    }
+
+    gtag("js", new Date());
+    gtag("config", "G-0Z09LP5XJ7");
+  };
+</script>
+
+<!-- Google tag (gtag.js) -->
+<script
+  async
+  src="https://www.googletagmanager.com/gtag/js?id=G-0Z09LP5XJ7"
```

Comment:
Presumably the code should G-E9YHJ0YHJY as given in the basecamp thread, this is the WNC code.